### PR TITLE
Checklist - Add Google sheets import instructions

### PR DESCRIPTION
**Pull request by kingthorin**
This PR adds Google sheets import instructions to the README in the checklist folder.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: checklist/README.md [hunk 1]
```diff
@@ -16,3 +16,19 @@ Contained in this folder is an Excel file which provides the following worksheet
...
+2. In another tab create a new [Google sheet](https://sheet.new).
+3. Copy the 'Direct Link' listed above.
+4. In your Google sheet select 'File > Import' from the main menu.
+5. In the 'Import file' dialog select the 'Upload' tab.
+6. Click 'Select a file from your device'.
+7. In the 'File Upload' dialog, paste the previously copied URL (from step 3) into the 'File name' field, click the 'Open' button.
+8. Wait a few seconds and a 'Import file' dialog should be displayed (choose your 'Import location', likely 'Replace spreadsheet'), then click the 'import data' button.
+9. You should now have a fully populated and functional Web Security Testing Guide Checklist Google sheet, with the four tabs as mentioned above.
+
+Note: These instructions are based on use of Mozilla Firefox (72.0.1) on Win 10, and Google sheets on 2020-01-12. Minor procedural differences may be necessary for other browsers, OSes, or future iterations of Google sheets.
```
Comment (victoriadrake) [inline checklist/README.md]: I'd place this before the list of instructions, which I started to follow on mobile (Android, Chrome) where they fail at step 4.

File: checklist/README.md [hunk 2]
```diff
@@ -16,3 +16,19 @@ Contained in this folder is an Excel file which provides the following worksheet
...
 
 SHA-256: 6ababbbc354c383179c56b0f4d41219694ae0cdbf90edb85506aff340e97dae9
+
+## Google Sheets Import
+
+The following instructions can be used to import the Checklist spreadsheet directly into a new Google sheet without having to save the doc locally first.
+
+1. Visit this page :+1:
+2. In another tab create a new [Google sheet](https://sheet.new).
+3. Copy the 'Direct Link' listed above.
```
Comment (victoriadrake) [inline checklist/README.md]: Properly, all the single quotes should be double quotes.

File: checklist/README.md (hunk 1 above)
Comment (kingthorin) [reply checklist/README.md]: Thanks

File: checklist/README.md (hunk 2 above)
Comment (kingthorin) [reply checklist/README.md]: Okay

**Pull request merged**